src/main/java/frc/robot/subsystems/Transfer.java

```diff
@@ -31,36 +34,23 @@ public Transfer() {
   public boolean isGamePieceCollected() {
     double current = transferMotor.getStatorCurrent().getValue();
     double desiredVelocity = prefTransfer.transferNoteVelocityTolerance.getValue();
-    return true;
-    // line 33 is temp NOT DONE
+    double belowCurrent = prefTransfer.transferGamePieceCollectedBelowAmps.getValue();
+    double aboveCurrent = prefTransfer.transferGamePieceCollectedAboveAmps.getValue();
+    if (current < belowCurrent && current > aboveCurrent
+        && Math.abs(transferMotor.getVelocity().getValue()) > Math.abs(desiredVelocity)) {
```

Comment:
For current and velocity detection we want the velocity to be below our set point. This says above

Reply:
You actually may want the velocity to be above because you get a current spike as you are speeding up. The point of the velocity check is to ensure you are not "detecting a game piece" as you get up to speed